### DISCUSSION TITLE
I have a "User postgres was denied access on the database .public" in docker when I use prisma?

### DISCUSSION THREAD

**Discussion by victor-dm-black** [prisma/prisma#11681] [Q&A] [answered]
I am not sure my problem depends on prisma but when I try to call function with prisma I had error 
Can't reach database server at `172.17.0.1`:`5432`
when I set "DATABASE_URL=postgresql: // user:123456a@172.17.0.1:5432/my_db?connection_limit=150&schema=public" 
or I got error 
User `postgres` was denied access on the database `.public`
If I set "DATABASE_URL=postgresql: // user:123456a@172.17.0.1:5432/my_db?connection_limit=150&schema=public&connect_timeout=1000".
Why did I ask it depends on prisma because I don't have any problems when I run prisma db push in docker or I run my ./temp/loadTariffs.js. 
```
RUN npm install
RUN npx prisma generate
RUN npx prisma db push --preview-feature

COPY --chown=node:node ./temp ./temp
RUN ./temp/loadTarrifs.js // I don't have any problems
COPY --chown=node:node . .

CMD [ "node", "index.js" ] I have an problem^
```

**Comment by victor-dm-black** [comment, marked as answer]
Need to set default network

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
